core/bases/src/main/java/com/chocolatecake/util/BindingAdapters.kt

```diff
@@ -145,6 +145,15 @@ fun <T> View.showWhenNoResult(list: List<T>?) {
     }
 }
 
+@BindingAdapter(value = ["app:hideWhenLoading"])
```

Comment:
this is better

fun View.hideWhenLoading(isLoading: Boolean?) {
   visibility = if (isLoading == true) {
    View.INVISIBLE
    } else {
      View.VISIBLE
    }
}